source/renderer/app/stores/NewsFeedStore.js

```diff
@@ -198,26 +198,48 @@ export default class NewsFeedStore extends Store {
     }
   };
 
+  @action setFakesNewsfeed = () => {
+    if (isDev) {
+      if (this.pollingNewsIntervalId) {
+        clearInterval(this.pollingNewsIntervalId);
+        this.pollingNewsIntervalId = null;
+      }
+      const rawNews = require('../config/news.dummy.json');
```

Comment:
What about moving this line to top of the file?

Reply:
@yakovkaravelov yeah that is the practice but I want to keep that inside the faked function for dev environment. Does that make sense to you?